tls/extensions/s2n_server_status_request.c

```diff
@@ -47,20 +47,7 @@ int s2n_server_status_request_recv(struct s2n_connection *conn, struct s2n_stuff
 
 /* Old-style extension functions -- remove after extensions refactor is complete */
 
-int s2n_server_extensions_status_request_send_size(struct s2n_connection *conn)
-{
-    if (s2n_server_can_send_ocsp(conn)) {
-        return 2 * sizeof(uint16_t);
-    }
-
-    return 0;
-}
-
-int s2n_server_extensions_status_request_send(struct s2n_connection *conn, struct s2n_stuffer *out)
-{
-    return s2n_extension_send(&s2n_server_status_request_extension, conn, out);
-}
-
+/* used in fuzz test */
 int s2n_recv_server_status_request(struct s2n_connection *conn, struct s2n_stuffer *extension)
```

Comment:
Are these comments necessary? Searching for occurrences of the method should return the fuzz tests just like the unit tests. What if we remove the fuzz test reference, but don't clean up this comment?

Reply:
Not strictly necessary. I'll go ahead and remove them.